### PR TITLE
enforce / reduce to two or fewer name components in patient name conversion utility with tests

### PR DESCRIPTION
**Pull request by sjswerdloff**
addresses #1137

### CODE REVIEW

**Comment by sjswerdloff** [comment]
got the commit wording wrong on first commit (that was the fix)

File: pymedphys/_utilities/patient.py
Comment (SimonBiggs) [inline pymedphys/_utilities/patient.py:36]:
```suggestion
    # TODO: In the future it would be nice to handle multiple names here
    patient_name = convert_patient_name_from_split(*patient_split[0:2])
```

**Review by SimonBiggs — APPROVED** [review]
Awesome, thanks @sjswerdloff :)

In the future it might be nice to handle multiple names. Not for this PR, so figured it'd be worth putting a `TODO` comment in-line.

**Comment by SimonBiggs** [comment]
Happy for you to merge anytime after approval, so feel free to bring this in once you're happy with it.

**Comment by SimonBiggs** [comment]
Don't bother waiting for "LGTM" before merge:

image

Anything it picks up really should be picked up by pylint, it's more just an independent run through.

**Pull request merged**
